FIO/local-volumes/README.md

```diff
@@ -95,3 +97,95 @@ injected into the Job.
 ```bash
 ~$ kubectl logs $POD
 ```
+
+
+## DBench FIO Tests
+
+### Local Volume with no replicas
+
+The script `dbench-job-generator-local-volume.sh` provisions a StorageOS Volume
+and a Pod on the same node and then will execute a series of FIOs (15s per FIO
+test) on the provisioned volume to measure StorageOS performance. It requires
+the StorageOS CLI running as a pod in the cluster.
+
+1. Generate the FIO job
+
+    ```bash
+    $ ./dbench-job-generator-local-volume.sh
+    ```
+
+    > It generates job manifest `local-volume-without-replica-fio.yaml` in `temp-local-fio/`
+
+1. On a new terminal while the script is running check the state of the
+   resources provisioned
+
+    ```bash
+    # Check that all the PVCs are provisioned
+    $ kubectl get pvc
```

Comment:
Having some sample outputs on the check commands would be quite useful for some users to know that they are on the right path. This would apply to the other check commands as well.